src/lightmap.cpp

```diff
@@ -1206,8 +1206,11 @@ float fastexp( float x )
         float f;
         int i;
     } u, v;
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wimplicit-int-float-conversion"
```

Comment:
Should probably be

```suggestion
#pragma clang diagnostic ignored "-Wimplicit-int-float-conversion"
```
or

```suggestion
#pragma GCC diagnostic ignored "-Wunknown-pragmas"
#pragma GCC diagnostic ignored "-Wimplicit-int-float-conversion"
```

Reply:
Great, since I opened a PR from someone else's repo, I can't edit it or append a commit in the web interface.